include/flamegpu/simulation/AgentVector_Agent.h

```diff
@@ -1,8 +1,8 @@
 #ifndef INCLUDE_FLAMEGPU_SIMULATION_AGENTVECTOR_AGENT_H_
 #define INCLUDE_FLAMEGPU_SIMULATION_AGENTVECTOR_AGENT_H_
 
-/**
```

Comment:
Not sure why you're removing the second `**`, I always put them back in.

Reply:
** makes it a docstring, not a multiline comment, which doxygen associates with the next namespace / class / mehtod it gets to, unless told otherwise. 

This is a file level comment, so not a docstring that should be assocated with a particular class nor namespace.